testing/suitesparse/APKBUILD

```diff
@@ -0,0 +1,60 @@
+# Contributor: Jakub Jirutka <jakub@jirutka.cz>
+# Maintainer: Jakub Jirutka <jakub@jirutka.cz>
+pkgname=suitesparse
+_pkgname=SuiteSparse
+pkgver=4.5.3
+pkgrel=0
+pkgdesc="A collection of sparse matrix libraries"
+url="http://faculty.cse.tamu.edu/davis/suitesparse.html"
+arch="x86 x86_64"
+license="GPL"
+depends_dev="openblas-dev"
+makedepends="$depends_dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="http://faculty.cse.tamu.edu/davis/$_pkgname/$_pkgname-$pkgver.tar.gz
+	0001-exclude-metis.patch
+	0002-remove-rpath.patch
+	"
+builddir="$srcdir/$_pkgname"
+
+prepare() {
+	cd "$builddir"
+
+	# OpenBLAS also provides LAPACK (?)
```

Comment:
openblas has a few changes to lapack, I don't think they're absolutely critical for performance but sometimes it can be more convenient to have the libblas and liblapack just in the same library (anything that is ever set up to use Intel MKL probably has to support this kind of setup). Some distros are very traditional about how they package separate blas and lapack libraries, maybe for the sake of switching them around with alternatives, but the threading api's in optimized blas/lapack implementations are all different so I don't think alternatives is really all that useful here.